revolut/__init__.py

```diff
@@ -602,7 +602,6 @@ class Order(_UpdateFromKwargsMixin):
     payments: str = ""
 
     def __init__(self, **kwargs):
-        kwargs.pop("payments")
```

Comment:
Do takich samo-poprawek fajne jest nadpisywanie historii. `git rebase -i` oraz `git push origin +nazwabrancha` do tego służą. Dopóki nie zmergowany, to wszystko gra i czyści commit log.

Ale to na przyszłość, bo merge już zrobiłem ;)